tracee/printer.go

```diff
@@ -105,28 +107,45 @@ func newEvent(ctx context, argsNames []string, args []interface{}) (Event, error
 }
 
 type tableEventPrinter struct {
-	tracee  *Tracee
-	out     io.Writer
-	err     io.Writer
-	verbose bool
+	tracee        *Tracee
+	out           io.Writer
+	err           io.Writer
+	verbose       bool
+	containerMode bool
 }
 
 func (p tableEventPrinter) Init() {}
 
 func (p tableEventPrinter) Preamble() {
 	if p.verbose {
-		fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-6s %-6s %-6s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID", "TID", "PPID", "RET", "EVENT", "ARGS")
+		if p.containerMode {
+			fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-11s %-11s %-11s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID/host", "TID/host", "PPID/host", "RET", "EVENT", "ARGS")
+		} else {
+			fmt.Fprintf(p.out, "%-14s %-16s %-12s %-12s %-6s %-16s %-5s %-5s %-5s %-16s %-20s %s", "TIME(s)", "UTS_NAME", "MNT_NS", "PID_NS", "UID", "COMM", "PID", "TID", "PPID", "RET", "EVENT", "ARGS")
```

Comment:
Using 5 chars for pid is not enough. Just ran Tracee and got pid: 875107 (just an example)

Reply:
did you customize your max-pid config? as far as I understand the common max is 32768

Reply:
no

Reply:
expanded to 7